### DISCUSSION TITLE
限定公開で投稿する際にOrganization記事として投稿したい

### DISCUSSION THREAD

**Discussion by stk2k** [increments/qiita-discussions#216] [要望・アイデア・不具合]
現状、初回で公開する場合はOrganization記事として投稿できますが、限定公開してから公開に変更する際にはOrganizationとして投稿のオプションはないように思います。
組織内で限定公開にしておいて固有URLを共有＆レビューし、OKなら公開するような運用フローにしたいのですが、限定公開で投稿時に記事をOrganization所属として投稿するようにできないでしょうか？

もしくは、下書きをOrganization記事として投稿できるようにして、組織のアカウントは下書きを見れる、みたいな方法でもいいのですが。

**Comment by tanaka-asami415** [comment]
@stk2k 

ご意見ありがとうございます！
実際、運営メンバーの間でも欲しいよねと話に挙がったことのある機能なので今後やっていきたいです！
ご不便をおかけしますが、しばらくお待ちいただければ幸いです🙏

**Comment by stk2k** [comment]
@tanaka-asami415 

開発作業、大変だと思いますがよろしくお願いいたします！
